crenata/abc/builder.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+from abc import ABC
+from typing import Any
+
+from discord import Embed
+
+
+class AbstractEmbedBuilder(ABC):
+    def __init__(self) -> None:
+        self.embed = Embed(color=5681003)
+
+    def follow_private_preference(self, school_name: str) -> str:
+        if self.private:
+            school_name = "비공개"
+        return school_name
```

Comment:
```suggestion
    def follow_private_preference(self, **kwargs: str) -> Dict[str]:
        if self.private:
            return {key: "비공개" for key, _ in kwargs.items()}
        return kwargs
```

이거 클래스가 AbstractEmbedBuilder니까 아예 일반화 해보는건 어떤가요?

Reply:
되면 Namespace로 반환받아도 괜찮을 것 같아요